.github/workflows/ci-frontend.yml

```diff
@@ -61,6 +61,13 @@ jobs:
             - name: Check if "schema.json" is up to date
               run: pnpm schema:build:json && git diff --exit-code
 
+            - name: Check toolbar bundle size
```

Comment:
We could add an `if` condition and only run this action if any of the `frontend/src/toolbar` files are modified, speeding up a bit the job.
Could use https://github.com/dorny/paths-filter